backend/src/resolvers.js

```diff
@@ -0,0 +1,67 @@
+const { UserInputError } = require('apollo-server');
+
+module.exports = {
+    Query: {
+      async posts(parent, args, { dataSources }) {
+        return await dataSources.postsApi.getPosts();
+      },
+      async users(parent, args, { dataSources }) {
+        return await dataSources.usersApi.getUsers();
+      }
+    },
+    User: {
+      async posts(parent, args , { dataSources }) {
+        const posts = await dataSources.postsApi.getPosts();
+        return posts.filter(post => post.author === parent.name);
+      }
+    },
+    Post: {
+      author(parent) {
+        return {
+          name: parent.author
+        };
+      }
+    },
+    Mutation:{
+      async write(parent, args, { dataSources }) {
+        let {
```

Comment:
```suggestion
        const {
```
Why is your linter not complaining about this?